app/wallet/service.js

```diff
@@ -95,6 +95,13 @@ export function formatTransaction(block, transaction, index) {
   const type = transaction.getTransactionType();
   let direction = "";
 
+
+  let debitAddresses = [];
+  transaction.getDebitsList().forEach((debit) => debitAddresses.push(debit.getPreviousAccount()));
```

Comment:
Since these aren't addresses, you should name this as "debitAccounts".

Reply:
Done...